homeassistant/components/androidtv/media_player.py

```diff
@@ -304,12 +310,17 @@ def adb_command(self, cmd):
         """Send an ADB command to an Android TV / Fire TV device."""
         key = self._keys.get(cmd)
         if key:
-            return self.aftv.adb_shell('input keyevent {}'.format(key))
+            self.aftv.adb_shell('input keyevent {}'.format(key))
+            self._adb_response = ''
+            return self._adb_response
 
         if cmd == 'GET_PROPERTIES':
-            return self.aftv.get_properties_dict()
+            self._adb_response = str(self.aftv.get_properties_dict())
+            return self._adb_response
 
-        return self.aftv.adb_shell(cmd)
+        response = self.aftv.adb_shell(cmd)
+        self._adb_response = str(response) if response else ''
```

Comment:
Is the else part for error reporting? Should errors also be reported somehow?

Reply:
`response` is either a string or `None`, so I'm handling both cases.